Dt node object details (#5207)
chef/automate: components/docs-chef-io/content/automate/chef_infra_server.md

{{< figure src="/images/automate/node-information-metadata.png" alt="Node Information & Metadata">}}

The next section of the page contains two options:

- Details
- Run List

##### Details

> Details section is to view or update the node environment. The drop down menu consists of the list of environment created in the chef server. You can select any one environment for the selected node.

Reviewer: ```suggestion
The **Details** section lets you view and update the node environment. The drop-down menu consists of the list of environments created in the chef server. You can select any one environment for the selected node.
```

{{< figure src="/images/automate/update-node-environment.png" alt="Update Node Environment">}}

##### Run List

> You can edit, expand or collapse the run list of a node.

Reviewer: ```suggestion
You can edit, expand or collapse a node's run list.
```

{{< figure src="/images/automate/run-list-of-a-node.png" alt="Run List of a Node">}}

> Chef Infra allows you to edit the run list of a node by following the steps below:

Reviewer: ```suggestion
Edit a run list by:
```

> 1. Select **edit** option. A popup window opens.

Reviewer: ```suggestion
1. Select **edit**. A popup window opens.
```

> 2. The left section the window contains the list of environment of the nodes. Select any one environment from the list.

Reviewer: ```suggestion
2. The left side of the window contains the list of environments. Select an environment from the list.
```

> 3. The right section of the window will show the current run list of the selected environment.

Reviewer: ```suggestion
3. The right side of the window will show the run list for the selected environment.
```

> 4. Select the run list you want for the specific environment and select **Create**.

Reviewer: ```suggestion
4. Select the run list for a specific environment.
5. Select **Create**.
```

> The popup window looks like as shown below:

Reviewer: ```suggestion
This opens a pop-up window similar to:
```

{{< figure src="/images/automate/edit-run-list.png" alt="Edit Run List">}}

> Chef Infra allows you to edit the run list of a node using a shortcut. Select the option **Edit Run List** from the ellipses icon of the specific node in the node list. Selecting the option will open the same popup window which can be used to edit the run list of a node.

Reviewer: ```suggestion
You can find the editing window directly by selecting **Edit Run List** from the options list ( {{< fontawesome class="fas fa-ellipsis-h" >}}) of a specific node in the node list. Selecting the option opens a pop-up for editing the run list.
```

Reviewer: I think that should work, but check the build

> The ellipses icon list looks like:

Reviewer: ```suggestion
```

> {{< figure src="/images/automate/edit-run-list-ellipses-icon.png" alt="Edit Run List Ellipses Icon">}}

Reviewer: This image isn't very helpful in understanding what the ellipses is. I think it is better to remove it.

> {{< figure src="/images/automate/edit-run-list-ellipses-icon.png" alt="Edit Run List Ellipses Icon">}}

Reviewer: ```suggestion
```

#### Manage Tags

> Chef Infra allows you to manage tags of the environment. You can add or remove multiple tags by:

Reviewer: ```suggestion
Chef Infra allows you to manage tags of the environment. You can add or remove multiple tags in two ways:
```

> - Selecting the node and add the tags from the text bar below the dropdown menu of environment.

Reviewer: ```suggestion
1. Selecting the node and add the tags from the text bar below the dropdown menu of environments.
```

> 

Reviewer: ```suggestion
```

> - OR, by simply selecting the **Manage Tags** option in the ellipses icon of a specific node. Selecting the Manage Tags option from the ellipses icon opens a popup window, where you can add multiple tags and click **Update Tags**. The popup window looks like as shown below:

Reviewer: ```suggestion
2.  Selecting **Manage Tags** from the options list ( {{< fontawesome class="fas fa-ellipsis-h" >}}) of a specific node. Add and remove tags in tag editor and select **Update Tags** to save your changes.
```

{{< figure src="/images/automate/manage-tags-from-ellipses-icon.png" alt="Manage Tags">}}

#### Reset a Node Key

> You can reset the key of a node by selecting the **Reset Key** option from the ellipses icon of a specific node in the node list. Selecting the option opens a warning popup specifying that the current key will no longer be accepted. Select **Reset Key** once again to confirm. The popup looks like:

Reviewer: I'm confused about this section. The "node key" isn't a concept in Manage. Should it be client key?

Author: I have updated the documentation

> You can reset the key of a node by selecting the **Reset Key** option from the ellipses icon of a specific node in the node list. Selecting the option opens a warning popup specifying that the current key will no longer be accepted. Select **Reset Key** once again to confirm. The popup looks like:

Reviewer: ```suggestion
Reset a node key by selecting **Reset Key** from the options list ( {{< fontawesome class="fas fa-ellipsis-h" >}}) of a specific node in the node list. Selecting **Reset Key** opens a warning that the current key will no longer be accepted. Select **Reset Key** once again to confirm. 
```
